_data/languages.json

```
@@ -19,6 +19,10 @@
     "title": "Русский",
     "id": "ru"
   },
+  {
+    "title": "Українська",
+    "id": "uk"
```

I'm not sure, but isn't it `uk-UA`?

```suggestion
    "id": "uk-UA"
```

---

ISO 639-1 code is `uk`